unstructured/file_utils/filetype.py

```diff
@@ -93,6 +93,11 @@ def __lt__(self, other):
     "image/jpeg": FileType.JPG,
     "image/png": FileType.PNG,
     "text/plain": FileType.TXT,
+    "text/x-csv": FileType.CSV,
+    "application/csv": FileType.CSV,
+    "application/x-csv": FileType.CSV,
+    "text/comma-separated-values": FileType.CSV,
+    "text/x-comma-separated-values": FileType.CSV,
```

Comment:
These will need to be excluded when we reverse the dict for `FILETYPE_TO_MIMETYPE`.

Reply:
Excluded! Though they're sad they didn't get an invite